BinaryEncoding.md

```diff
@@ -81,7 +81,7 @@ Yes:
   - A table (sorted by offset) containing, for each type which has opcodes:
     + A standardized string literal [type name](AstSemantics.md#local-and-memory-types).
       The index of a type name in this table is referred to as a type ID
-    + 32-bit offset of its opcode table within the section
+    + 64-bit offset of its opcode table within the section
```

Comment:
I like it for some things because then I don't have to specify 32 or 64 bit sizes for something that might change in the future. (Like storage of data from a profiler where values may reasonably be either 32 or 64 bit depending on the target or the nature of the data being sampled.) The compression is a secondary concern in that sort of case.